runtime/Java/src/org/antlr/v4/runtime/atn/LexerATNSimulator.java

```diff
@@ -82,8 +82,6 @@ protected void reset() {
 	@NotNull
 	protected final SimState prevAccept = new SimState();
 
-	public static int match_calls = 0;
```

Comment:
📝 When I do eventually update this file to a version that includes this change, I'm going to revert this line and mark the field deprecated instead of removing it altogether. This will serve two purposes:

1. Users who reference the field will still load correctly.
2. Users who need the field can derive from `LexerATNSimulator` to add their own call to `match_calls++` if needed to preserve the original behavior.

Reply:
Gotcha. Thank you, Sam ;-)